app_dart/lib/src/service/config.dart

```diff
@@ -347,19 +347,23 @@ class Config {
 
   KeyHelper get keyHelper => KeyHelper(applicationContext: context.applicationContext);
 
-  // Default number of commits to return for benchmark dashboard.
+  /// Default number of commits to return for benchmark dashboard.
   int /*!*/ get maxRecords => 50;
```

Comment:
While you're here, remove the `/*!*/`?

Reply:
Autosubmit happened - will address in a followup